### DISCUSSION TITLE
Can creation of new langauges be limited to a minimum percent of translation ?

### DISCUSSION THREAD

**Discussion by strk** [WeblateOrg/weblate#10206] [Q&A] [answered]
We configured a self-hosted weblate to directly push commits to our project's repository and a few days ago a new language directory was pushed with NO strings translated. Could we prevent this by requiring a minimum percentage of translated strings before a new language is pushed to the repository ?

**Comment by nijel** [comment, marked as answer]
Not currently, see https://github.com/WeblateOrg/weblate/issues/3436
